src/Dashboard/Data/FunctionIndexReader.cs

```diff
@@ -70,7 +70,8 @@ public DateTimeOffset GetCurrentVersion()
             return _versionMapper.GetVersion(blob.Metadata);
         }
 
-        public IResultSegment<FunctionIndexEntry> Read(int maximumResults, string continuationToken)
+        // Classic logging ignores hostName filter. 
```

Comment:
Yes.

---
In reply to: [84854250](https://github.com/Azure/azure-webjobs-sdk/pull/884#discussion_r84854250) [](ancestors = 84854250)